include/client/libcurve.h

```diff
@@ -352,6 +406,50 @@ int GetClusterId(char* buf, int len);
 namespace curve {
 namespace client {
 
+struct OpenFlags {
```

Comment:
I think we can just delete it, and use a `int` to store the permissions and other flags, like the second paramater of `open`, https://man7.org/linux/man-pages/man2/open.2.html